test-suite-ex/src/test/scala/org/scalajs/testsuite/utils/Platform.scala

```diff
@@ -10,6 +10,7 @@ package org.scalajs.testsuite.utils
 /** This is a partial copy of the implementation in the testSuite */
 object Platform {
 
+  def executingInNodeJS: Boolean = sysProp("nodejs")
   def typedArrays: Boolean = sysProp("typedarray")
 
   private def sysProp(key: String): Boolean =
```

Comment:
That would defeat the purpose of all the other tests in `testSuiteEx`, which fail to fail if they are part of the larger codebase of the test suite. So no, can't do that.